src/assets/css/styles.css

```diff
@@ -1,262 +1,277 @@
+    right: 0;
+    height: 100vh;
+    top: 8vh;
+    background-color: var(--footerDarkBg);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+    transform: translate(100%);
+    z-index: -1;
+  }
+
+  .burger {
+    display: block;
+    margin-right: 16rem;
+  }
+
+  .nav-list li {
+    opacity: 0;
+  }
 }
 
 @media screen and (max-width: 767px) and (max-width: 667px) {
-    .nav-list {
-        top: 9vh;
-    }
+  .nav-list {
+    top: 9vh;
+  }
 
-    .burger {
-        display: block;
-        margin-right: 13rem;
-    }
+  .burger {
+    display: block;
+    margin-right: 13rem;
+  }
 }
 
 /* Hamburger Menu Logic from Javascript */
 .active {
-    transform: translateX(0%);
+  transform: translateX(0%);
 }
 
-.toggle .line-one { 
-    transform: rotate(-45deg) translate(-5px,5px);
+.toggle .line-one {
+  transform: rotate(-45deg) translate(-5px, 5px);
 }
 
 .toggle .line-two {
-    opacity: 0;
+  opacity: 0;
 }
 
 .toggle .line-three {
-        transform: rotate(45deg) translate(-5px,-5px);
+  transform: rotate(45deg) translate(-5px, -5px);
 }
 
 @keyframes navLinkFade {
-    from {
-        opacity: 0;
-        transform: translateX(50px);
-    }
+  from {
+    opacity: 0;
+    transform: translateX(50px);
+  }
 
-    to {
-        opacity: 1;
-        transform: translateX(0px);
-    }
+  to {
+    opacity: 1;
+    transform: translateX(0px);
+  }
 }
 
 /* Button  */
 .buttons {
-    margin-top: 60px;
-    text-align: center;
+  margin-top: 60px;
+  text-align: center;
 }
 
 .buttons a {
-    margin: 10px 40px;
+  margin: 10px 40px;
 }
 
 .vision-content a {
-    transition: all 0.5s ease-in-out;
+  transition: all 0.5s ease-in-out;
 }
 
 .vision-content a span {
-    padding-left: 1.5px;
-    transition: all 0.4s ease-in-out;
+  padding-left: 1.5px;
+  transition: all 0.4s ease-in-out;
 }
 
 .vision-content a:hover span {
-    transform: rotate(90deg);
+  transform: rotate(90deg);
 }
 
 /* Projects */
 #projects {
-    height: 400px;
+  height: 400px;
 }
 
 .projects {
-    margin: 0 auto;
-    max-width: 1200px;
+  margin: 0 auto;
+  max-width: 1200px;
 }
 
 .projects h2 {
-    text-align: center;
-    margin: 50px 0 20px;
+  text-align: center;
+  margin: 50px 0 20px;
 }
 
 .project {
-    padding: 40px;
-    background: #fff url(../images/under-construction.png) no-repeat center center/cover;
+  padding: 40px;
+  background: #fff url(../images/under-construction.png) no-repeat center
+    center/cover;
 }
 
 .project p {
-    font-size: 16px;
-    line-height: 1.6;
+  font-size: 16px;
+  line-height: 1.6;
 }
 
 .project div {
-    margin-top: 20px;
-    display: flex;
-    width: 100%;
-    justify-content: flex-end;
+  margin-top: 20px;
+  display: flex;
+  width: 100%;
+  justify-content: flex-end;
 }
 
 .project div span {
-    margin: 0 5px;
-    color: var(--secondaryBtn);
-}
\ No newline at end of file
+  margin: 0 5px;
+  color: var(--secondaryBtn);
+}
+
+.flex-col-2 {
```

Comment:
I use css columns to make the project cards responsive (#247)

Reply:
Yeah, this is nice. Actually, I've not used it much. That's why didn't came with this approach.
I will remove my commit (on responsiveness) and we can go ahead with your changes